pkg/astrolabe/constants.go

```diff
@@ -1,5 +1,7 @@
 package astrolabe
 
 const (
-	PvcPEType = "pvc"
+	PvcPEType        = "pvc"
+	ParaVirtPvPEType = "paravirt-pv"
```

Comment:
Ideally, astrolabe core code should not be aware of paravirt PE type. But, we can revisit and refine it later.

Reply:
sure.